routing.go

```diff
@@ -114,13 +112,13 @@ func (r *routelet) watchRoutingInfo(ctx context.Context, env env, group string)
 		routingInfo, newVersion, err := env.GetRoutingInfo(ctx, group, version)
 		if err != nil {
 			// TODO(mwhittaker): Handle errors more gracefully.
```

Comment:
I was considering a fix for this, but I'm not sure how the errors should be handled. Should an error here cause `watchRoutingInfo` to exit? Or is there some other way to handle these?

Reply:
Hmm, yeah this is tricky. I don't think an error should cause `watchRoutingInfo` to exit. The RPC to get routing info may experience a network error, for example, that should be retried. But, I also don't know what to do besides just logging and retrying. We should probably---here and everywhere else in the code---be more careful about which errors are retriable and which aren't. Maybe just leave it is as is for now since I think it's a more complicated fix.